model_compression_toolkit/core/common/quantization/quantization_params_generation/lut_kmeans_params.py

```diff
@@ -62,11 +69,12 @@ def lut_kmeans_tensor(tensor_data: np.ndarray,
     kmeans = KMeans(n_clusters=n_clusters)
     axis_not_channel = [i for i in range(len(tensor_data.shape))]
     axis_not_channel.remove(channel_axis)
-    if per_channel:
-        thresholds_per_channel = np.max(np.abs(tensor_data), axis=tuple(axis_not_channel), keepdims=True)
-    else:
-        thresholds_per_channel = np.max(np.abs(tensor_data), keepdims=True)
-    thresholds_per_channel = np.power(2.0, np.ceil(np.log2(thresholds_per_channel)))
+
+    threshold_selection_tensor = symmetric_selection_tensor if is_symmetric else power_of_two_selection_tensor
+    thresholds_per_channel = threshold_selection_tensor(tensor_data, p, n_bits, per_channel,
+                                                        channel_axis, n_iter, min_threshold,
+                                                        qc.QuantizationErrorMethod.NOCLIPPING)[THRESHOLD]
```

Comment:
Why not use the given quant_error_method instead of no clipping?

Reply:
MSE's may have unexpected bevaiour, so as it is the default, we decided on fixing it to NC for now